### PR TITLE
* Edits & comment out unused code

### PR DESCRIPTION
**Pull request by Wagnerp**
* Comment out unused code to reduce compiler warnings

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs [hunk 1]
```diff
@@ -120,7 +120,7 @@ public Color ContextColor
             set
             {
                 // We never allow a null or transparent color
-                if ((value == null) || (value == Color.Transparent))
+                if (/*(value == null) ||*/ (value == Color.Transparent)) // 'color' is never 'null' CS8073
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs:123]: **Do not do this** for public setters, the compiler can never know how these are going to be used in the "Wild"
Just add the warning to the `.editorconfig`

**Comment by Wagnerp** [comment]
A bit lost here... 

image

**Comment by Smurf-IV** [comment]
> 
> 
> A bit lost here...
> 

That view is supposed to be helpful.. I have never found it to be more confusing..
Goto "Open with (Right click on file) and use the default text editor.. It's a lot easier for a developer to use then the unhelpful management view

File: Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs:123]: Added to `.editorconfig`

**Comment by Wagnerp** [comment]
image

**Pull request merged**